pipeline/data_merging/variant_equivalence.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from collections import defaultdict
+
+
+def calculate_edited_seq(chr, pos_chr, ref, alt, seq_provider):
+    seq, seq_start = seq_provider.get_seq_with_start(chr)
+
+    pos_seq = int(pos_chr) - seq_start - 1
+
+    assert pos_seq >= 0, "v1 positions is below the reference"
+    assert pos_seq + len(ref) <= len(seq), "v1 position is above the reference"
+
+    # TODO: fix. tests fail, as it seems illegal variants are constructed
```

Comment:
if this is still a problem it should be resolved before merge

Reply:
I've fixed the tests now.